docs/styling/THEME.md

```diff
@@ -408,3 +408,113 @@ Here is an example of how your files architecture could look:
 >>>> **my-component-pres.style.scss**: component style - import style.theme.scss\
 >>>> **my-component-pres.style.theme.scss**: variables - import app-styling.scss\
 >>>> ...
+
+## Technical structure (advance)
+
+The function `apply-theme` expects a specific structure of [SCSS Map](https://sass-lang.com/documentation/values/maps/). To facilitate the generation of the latter, the function `meta-theme-to-otter` converts an [Angular Material Theme](https://material.angular.io/guide/theming) into a compatible structure.
+
+The theme structure object should respect the following Schema:
+
+```json
+{
+  "$ref": "#/definitions/varNode",
+
+  "definitions": {
+    "varNode": {
```

Comment:
to be honnest, it is a bit difficult to separate name that are just placeholders from actual expected properties

Reply:
I take all advice you can have on it :)

Reply:
Maybe between "[ ]" ?
This is just an idea but something like 
"[yourVarNode]" : { ... ?

Reply:
So you suggest to not go with JSON Schema anymore?

Reply:
yeah you are right, I guess it is not supported by the structure in general.